vm/systemSmartContracts/delegation.go

```diff
@@ -1427,24 +1495,27 @@ func (d *delegation) withdraw(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 		if totalUnBonded.Cmp(actualUserUnBond) > 0 {
 			unBondedFromThisFund := big.NewInt(0).Sub(totalUnBonded, actualUserUnBond)
 			fund.Value.Sub(fund.Value, unBondedFromThisFund)
+			//TODO: If saveFund will fail, then the save done in method resolveUnStakedUnBondResponse -> saveDelegationStatus will remain saved. Is this ok?
```

Comment:
nothing will remain. Any error invalidates all previous changes. save is done in a cache - cache is invalidated if returnCode != vmcommon.Ok

Reply:
ok